runtime/storage_test.go

```diff
@@ -3403,34 +3403,13 @@ func TestRuntimeStorageIteration(t *testing.T) {
             }
         `))
 
-		newRuntimeInterface := func() (Interface, *[]Location) {
-			programs := map[Location]*interpreter.Program{}
-
-			var programStack []Location
-
-			runtimeInterface := &testRuntimeInterface{
+		newRuntimeInterface := func() Interface {
+			return &testRuntimeInterface{
 				storage: ledger,
 				getSigningAccounts: func() ([]Address, error) {
 					return []Address{address}, nil
 				},
 				resolveLocation: singleIdentifierLocationResolver(t),
-				getProgram: func(location Location) (*interpreter.Program, error) {
```

Comment:
The `getProgram` and `setProgram` really only checked the balancing of the `GetProgram` and `SetProgram` calls. Given that this is no longer possible, remove those assertions